src/app/pages/project/milestone/print-milestone/print-milestone.component.ts

```diff
@@ -0,0 +1,388 @@
+    /* tslint:enable:max-line-length */
+    this.doc = new jsPDF();
```

Comment:
please, create a new PdfDocService class and put there related methods like
getDoc(image){
 this.doc = new jsPDF();

    this.doc.addImage(imgData, 'JPEG', 15, 15, 25, 25);
    this.doc.setFontSize(20);
    this.doc.setFontSize(10);
    this.doc.setTextColor(168, 168, 168);
    this.doc.text(42, 33, 'Aquality Tracking');
    this.doc.setTextColor(0, 0, 0);

    this.doc.setDrawColor(168, 168, 168);
    this.doc.setLineWidth(0.1);
    this.doc.line(72, 30, 72, 62);

    this.doc.setFontSize(10);
    this.doc.text(170, 15, new Date().toDateString());
    return doc
}

and here then use like this.doc = service.getDoc(image)